types/webextension-polyfill/namespaces/tabs.d.ts

```diff
@@ -1204,7 +1189,7 @@ export namespace Tabs {
          * @param details Details of the script to run.
          * @returns Called after all the JavaScript has been executed.
          */
-        executeScript(details: ExtensionTypes.InjectDetails): Promise<any[]>;
+        executeScript(details: ExtensionTypes.InjectDetails): Promise<unknown[]>;
```

Comment:
Same as above.

Reply:
Same reason: only the caller knows what type it's supposed to be.